acquire/acquire.py

```diff
@@ -404,11 +404,15 @@ def get_spec_additions(cls, target):
 @local_module
 class WinRDPSessions(Module):
     DESC = "Windows Remote Desktop session information"
-    SPEC = [
-        ("command", (["qwinsta", "/VM"], "win-rd-sessions")),
-    ]
     EXEC_ORDER = ExecutionOrder.BOTTOM
 
+    @classmethod
+    def get_spec_additions(cls, target):
+        qwinsta = subprocess.run(["where", "qwinsta.exe"], capture_output=True, text=True).stdout.rstrip("\n")
```

Comment:
Technically it could return multiple lines if the utility is present in multiple dirs in `%path%` and current dir.